cli/src/command/account.rs

```diff
@@ -328,6 +331,50 @@ pub async fn claim_command(account_handle: &AccountHandle, output_id: Option<Str
     Ok(())
 }
 
+/// `claimable-outputs` command
+pub async fn claimable_outputs_command(account_handle: &AccountHandle) -> Result<(), Error> {
+    let balance = account_handle.balance().await?;
+    for output_id in balance
+        .potentially_locked_outputs()
+        .iter()
+        .filter_map(|(output_id, unlockable)| unlockable.then_some(output_id))
+    {
+        if let Some(output_data) = account_handle.get_output(output_id).await {
+            let output = output_data.output;
+            let kind = match output {
+                Output::Nft(_) => "Nft",
+                Output::Basic(_) => "Basic",
+                _ => unreachable!(),
+            };
+            println_log_info!("{output_id} ({kind})");
+
+            if let Some(native_tokens) = output.native_tokens() {
+                println_log_info!("  - native token amount:");
+                native_tokens.iter().for_each(|token| {
+                    println_log_info!("    + {} ({})", token.amount(), token.token_id());
```

Comment:
Why a + here? :trollface: 
Also maybe switch it around so it matches the output?
```suggestion
                println_log_info!("  - amount | native token:");
                native_tokens.iter().for_each(|token| {
                    println_log_info!("    - {} ({})", token.amount(), token.token_id());
```

Reply:
`+` is psychology better than `-`. Trust me on this! :trollface:

Reply:
Then why not also have it for the amount and other things? :trollface: